From 8a884199dcabf3a92b66360f782139472a4694d2 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Fri, 16 Aug 2019 09:29:21 +0000
Subject: [PATCH] arm64: dts: rockchip: px30/rk3326: add rng as hardware random
 pool input

Sometimes, the random pool initialization is too slow in reocvery mode,
use hw_random to speed up it.

Change-Id: Iabc37548ef03e60dca9707126750629efafbfe6d
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-robot.dtsi   | 4 ++++
 arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/px30-robot.dtsi b/arch/arm64/boot/dts/rockchip/px30-robot.dtsi
index abe1b1b73376..a884a8e7fdcd 100644
--- a/arch/arm64/boot/dts/rockchip/px30-robot.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-robot.dtsi
@@ -53,6 +53,10 @@
 	/delete-node/ opp-768000000;
 };
 
+&rng {
+	status = "okay";
+};
+
 &soc_thermal {
 	trips {
 		threshold: trip-point-0 {
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
index 3a754df8866c..de7483ab3f75 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
@@ -107,6 +107,10 @@
 	};
 };
 
+&rng {
+	status = "okay";
+};
+
 &video_phy {
 	status = "okay";
 };
-- 
2.35.3

